etc/parameters/default.json

```diff
@@ -328,7 +328,8 @@
     "goal_post_obstacle_radius": 0.2
   },
   "role_assignment": {
-    "forced_role": null
+    "forced_role": null,
+    "keeper_replacementkeeper_switch_time": { "nanos": 0, "secs":  8}
```

Comment:
Based on the behavior sim file this should at least be 12, I recommend changing it to that and then doing a real life test.

Reply:
Done